packages/autorest.typescript/test/rlcIntegration/generated/dpgCustomization/package.json

```diff
@@ -45,7 +45,7 @@
     "@azure/core-rest-pipeline": "^1.5.0",
     "@azure/logger": "^1.0.0",
     "tslib": "^2.6.2",
-    "@azure/core-lro": "3.0.0",
+    "@azure/core-lro": "^3.0.0",
```

Comment:
Is it correct for autorest generated rlc to use core-lro v3 as a dependency? @MaryGao

Reply:
yes, this is correct.
- RLC from swagger/typespec: v3
- HLC from swagger: v2
- Modular from typespec: v3